lib/elixir/test/elixir/fixtures/dialyzer/remote_call.ex

```diff
@@ -24,10 +24,6 @@ defmodule Dialyzer.RemoteCall do
 
   def mod_var() do
     module = Hello
-    module.fun
-  end
-
-  def mod_var(module) when is_atom(module) or is_atom(elem(module, 0)) do
```

Comment:
A few lines above there is a `@dialyzer {:no_match, [map_var: 0, mod_var: 0, mod_var: 1]}` and I believe `mod_var/1` should be removed from the list 🙂